src/main/webapp/app/components/Header.tsx

```diff
@@ -234,6 +234,10 @@ class Header extends React.Component<IHeaderProps> {
                     <AccountMenu
                       isAuthenticated={this.props.isUserAuthenticated}
                       isAdmin={this.props.isAdmin}
+                      hideAccountText={
+                        this.props.windowStore.isLargeScreen &&
+                        !this.props.windowStore.isXLscreen
```

Comment:
Should this just be `!this.props.windowStore.isXLscreen`?

Reply:
Yeah, I think that is a better condition. I just changed the prop to `showAccountText` and the condition to `isXLscreen`.